tensorboard/compat/proto/layout.proto

```diff
@@ -0,0 +1 @@
+../../plugins/custom_scalar/layout.proto
```

Comment:
@nfelt what's your thought on symlinks here?

Reply:
I still don't see any reason why the plugin protos need to be reparented under tensorboard/compat/proto.